lib/game.rb

```diff
@@ -0,0 +1,113 @@
+require 'pry'
+class Game
+  attr_accessor :board, :player_1, :player_2, :last_turn
+  WIN_COMBINATIONS = [
+   [0,1,2],
+   [3,4,5],
+   [6,7,8],
+   [0,3,6],
+   [1,4,7],
+   [2,5,8],
+   [0,4,8],
+   [6,4,2]
+  ]
+
+  def initialize(player_1 = Player::Human.new("X"), player_2 = Player::Human.new("O"), board = Board.new)
+   @board = board
+   @player_1 = player_1
+   @player_2 = player_2
+  end
+
+  def ai_vs_ai
+    @player_1 = Player::Computer.new("X")
+    @player_2 = Player::Computer.new("O")
+  end
+
+  def ai_vs_human(choice) ## -->
+    if  choice == "X" || choice == "x"
```

Comment:
You could also use [`downcase`](http://ruby-doc.org/core-2.2.4/String.html#method-i-downcase) here.
`if choice.downcase == "x"`